ape_etherscan/client.py

```diff
@@ -183,16 +183,24 @@ def _request(
         headers = headers or self.DEFAULT_HEADERS
         for i in range(self._retries):
             response = self.session.request(
-                method.upper(), self.base_uri, headers=headers, params=params, data=data
+                method.upper(),
+                self.base_uri,
+                headers=headers,
+                params=params,
+                data=data,
+                timeout=1064,
```

Comment:
weird timeout choice

Reply:
give one that you find less weird plz

Reply:
i made less weird but gimme 1 if you still dont like it.

Reply:
That works, was more curious if there was any special meaning to 1064 that needed to be documented